model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -94,8 +94,22 @@ class ReverseChannelsPropagationDown(BackReplacementPattern):
 
         'Shape': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_shape(node, rc),
         'ShapeOf': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_shape(node, rc),
+        'Pad': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_pad(node, rc),
     }
 
+    @staticmethod
+    def pass_rc_through_pad(node: Node, reverse_channels: Node):
```

Comment:
nit: we can call it passthrough and if we need to add other operations here -- we could use it